python/tvm/relay/frontend/onnx.py

```diff
@@ -1081,6 +1081,16 @@ class Or(Elemwise):
     def _impl_v7(cls, inputs, attr, params):
         return _op.logical_or(inputs[0], inputs[1])
 
+class Expand(OnnxOpConverter):
+    """ Operator converter for Expand.
+    """
+
+    @classmethod
+    def _impl_v8(cls, inputs, attr, params):
+        shape_data = params[inputs[1].name_hint]
```

Comment:
Can you please use something similar to `Reshape`:

```
if get_name(inputs[1]) in params:
    shape_data = params[inputs[1].name_hint]
else:
    shape = infer_value_simulated(inputs[1], params)
```

Reply:
I fixed them by following `Reshape`.